salt/metalk8s/addons/dex/deployed/chart.sls

```diff
@@ -60,6 +61,10 @@ stringData:
       redirectURIs:
       - "{% endraw -%}https://{{ grains.metalk8s.control_plane_ip }}:8443/grafana/login/generic_oauth{%- raw %}"
       secret: 4lqK98NcsWG5qBRHJUqYM1
+    - id: external-oidc-ui
+      name: {% endraw -%}{{ dex.spec.externalIDP.staticClient.name }}{%- raw %}
+      redirectURIs:   {% endraw -%}{{ dex.spec.externalIDP.staticClient.redirectURIs | tojson }}{%- raw %}
```

Comment:
Weird those triple space, but never mind it's not related to change of this PR

```suggestion
      redirectURIs: {% endraw -%}{{ dex.spec.externalIDP.staticClient.redirectURIs | tojson }}{%- raw %}
```

Reply:
Likely from the render script.
Let me fix it

Reply:
Going to backport the changes maybe later!